<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuEmpleados</class>
 <widget class="QWidget" name="MenuEmpleados">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>639</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Menu Empleados</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>561</width>
     <height>461</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>3</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Agregar</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>481</width>
       <height>371</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLineEdit" name="inputCodigoEmpleado">
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="placeholderText">
         <string>Codigo</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="inputNombreEmpleado">
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="placeholderText">
         <string>Nombre</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="inputUsuarioEmpleado">
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="placeholderText">
         <string>Usuario/Nickname</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="inputContraseniaEmpleado">
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="placeholderText">
         <string>Contraseña/Password</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="inputTipoEmpleado">
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
        <property name="placeholderText">
         <string>Supervisor &quot;S&quot; o  Normal &quot;N&quot;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="botonAceptarAgregaEmpleado">
        <property name="text">
         <string>Aceptar</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Modificar</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>371</width>
       <height>292</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>6</number>
        </property>
        <item>
         <widget class="QPushButton" name="botonBuscarModEmpleados">
          <property name="text">
           <string>Buscar</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLineEdit" name="inputCodigoEmpleadoMod">
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="placeholderText">
             <string>Codigo</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="inputNombreEmpleadoMod">
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="placeholderText">
             <string>Nombre</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="inputUsuarioEmpleadoMod">
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="placeholderText">
             <string>Usuario/Nickname</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="inputContraseniaEmpleadoMod">
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="placeholderText">
             <string>Contraseña/Password</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="inputTipoEmpleadoMod">
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="placeholderText">
             <string>Supervisor &quot;S&quot; o  Normal &quot;N&quot;</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="botonGuardarModEmpleados">
          <property name="text">
           <string>Guardar</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
    <widget class="QTableWidget" name="tablaModificarEmpleados">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>310</y>
       <width>531</width>
       <height>121</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Mostrar">
    <attribute name="title">
     <string>Mostrar</string>
    </attribute>
    <widget class="QTableWidget" name="tablaEmpleados">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>521</width>
       <height>381</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="botonActualizarEmpleados">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>410</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Actualizar</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Eliminar</string>
    </attribute>
    <widget class="QWidget" name="layoutWidget_5">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>471</width>
       <height>351</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>6</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLineEdit" name="inputCodigoEmpleadoEliminar">
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="placeholderText">
           <string>Codigo</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="inputNombreEmpleadoEliminar">
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="placeholderText">
           <string>Nombre</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="inputUsuarioEmpleadoEliminar">
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="placeholderText">
           <string>Usuario/Nickname</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="inputContraseniaEmpleadoEliminar">
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="placeholderText">
           <string>Contraseña/Password</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="inputTipoEmpleadoEliminar">
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="placeholderText">
           <string>Supervisor &quot;S&quot; o  Normal &quot;N&quot;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="botonEliminarModEmpleados">
        <property name="text">
         <string>Eliminar</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
